CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Changes since v2.6
 - Reminder email about open applications to all handlers. The emails can
   be sent by calling `/api/email/send-handler-reminder` on a cron schedule
   or similar. The API requires an API key. (#1611)
+- Reminder email about applications with open review requests to reviewers.
```

Comment:
I have a problem with the terminology here. The difference between 'reviewer' and 'handler' is probably not obvious to a user that is not completely familiar with REMS before. Also, offhand I would assume that when an application is submitted, it would open a review request for the handlers, but apparently the semantics of 'review request' is such that it is opened only if the original handler of the application explicitly makes a review request to someone who is not marked as a handler in the workflow.

I think those semantics are unnecessarily complicated, but I would need to think a bit to come up with better alternatives.

(For what it's worth, when looking in Github at the issue which this patch closes, I didn't quite understand what review request refers to, so I'm pretty sure it could be likewise confusing for an average user.)

Reply:
There are "Request review" and "Request decision" buttons in the UI, so they should be terms that the users are acquainted with.